app/commands/git/sync_concept_authors.rb

```diff
@@ -15,7 +15,7 @@ def call
         # This is required to remove authors that were already added
         concept.reload.update!(authors: authors)
 
-        # TODO: consider what to do with missing authors
+        # TODO: (Optional): consider what to do with missing authors
```

Comment:
I don't think this is blocking, as we could theoretically scan the logs to find out. Also, nothing bad happens, it's just that those missing authors won't show up as contributors on the website.